config/string_list.go

```diff
@@ -74,3 +74,16 @@ func (sl StringList) String() string {
 func IsStringList(s string) bool {
 	return strings.Contains(s, stringListDelim)
 }
+
+func CompactStringList(sl StringList) StringList {
```

Comment:
Thanks, fixed that. The functions in `interpolate_funcs.go` were also ordered alphabetically, I had not noticed - fixed that too.